check_raid.pl

```diff
@@ -559,15 +558,41 @@ sub program_names {
 
 sub commands {
 	{
-		'status' => ['-|', '@CMD'],
+		'status' => [>&2', '@CMD'],
```

Comment:
On 16.02.2015 15:37, noblemtw wrote:
> I think ‘>&2’ is actually correct

but that's the thing, it is not what you describe here. it's with one 
apostrophe only.

-- 
glen